src/Assignment4/CardComponent/index.js

```diff
@@ -5,14 +5,21 @@ import "./styles-card.css";
 
 const CardComponent = ({
   temmatesDetails: { imageSource, name, location, company, followers },
-}) => (
-  <div className="cardWrapper">
-    <img src={imageSource} className="imageClass" alt={userIcon} />{" "}
-    <div className="information-class">Name : {name}</div>
-    <div className="information-class">Place: {location}</div>
-    <div className="information-class">Company: {company}</div>
-    <div className="information-class">Followers: {followers}</div>
-  </div>
-);
+}) => {
+  return (
+    <div className="cardWrapper">
+      <img
+        key={imageSource}
+        src={imageSource}
+        className="imageClass"
+        alt={userIcon}
+      />{" "}
```

Comment:
remove that comment {" "}. It's from long ages in your code. Joking!

Reply:
removed. actually prettier brings it for me